### DISCUSSION TITLE
inconsistency of Chinese localization in recent sub editor layer update

### DISCUSSION THREAD

**Discussion by Cyana-ir** [FakeFishGames/Barotrauma#13796] [Bug reports]
From changelog.txt, we known that: "Groups are now called "layers", which reflects their common uses better than "groups"." .
But... the localizations for Chinese (both Simplified and Traditional) still keep the name for "groups" in some parts in sub editor, which is inconsistent.
![EnVersionInV1 3](https://github.com/FakeFishGames/Barotrauma/assets/106088471/5baf71e9-9ce9-4109-bdf8-29fdfcbd9010)
![SCVersionInV1 3](https://github.com/FakeFishGames/Barotrauma/assets/106088471/d117c8d1-1f82-4b2b-927c-b5793e86c91c)
![EnVersionInV1 4](https://github.com/FakeFishGames/Barotrauma/assets/106088471/d2701061-1053-4b41-99e4-f04c95fd6164)
![SCVersionInV1 4](https://github.com/FakeFishGames/Barotrauma/assets/106088471/906fdb4c-3b4a-4200-92f9-5db1d4555e86)
For Fig 1,2, they are screenshots In v1.3.0.4， "groups" = “群组”
But for Fig 3,4, they are screenshots in v1.4.4.1, "layers" on new buttons equal to "层级" , "layers" on old buttons still equal to "群组"

**Comment by Cyana-ir** [comment]
After check the text files, here are localizations having problems:
<editor.layer>群组</editor.layer>
<editor.layer.movetolayer>发送选择到群组组...</editor.layer.movetolayer>
<editor.layer.createlayer>从选择中创建新群组组</editor.layer.createlayer>
<editor.layer.selectall>选择这些群组中的所有项目</editor.layer.selectall>
<editor.layer.nolayer>没有群组</editor.layer.nolayer>
<editor.layer.newlayer>新群群组</editor.layer.newlayer>
<editor.layer.renamelayer>重命名群组</editor.layer.renamelayer>
<editor.layer.deletelayer>删除群组</editor.layer.deletelayer>
<editor.layer.button>管理群组</editor.layer.button>
<editor.layer.tooltip>切换、创建、删除和重命名群组</editor.layer.tooltip>

**Comment by Regalis11** [comment]
Thank you for the report! I've created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13804) and reported this to the localization team.

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/14b11e86754a23dc96f2d1bd687dc8a1425fe3d3 (commit in our private repo), hopefully coming up in the next hotfix!
